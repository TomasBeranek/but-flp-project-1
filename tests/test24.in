A,B
a,b,c
A10
A->aB
A->cB
B->bB
B->#
