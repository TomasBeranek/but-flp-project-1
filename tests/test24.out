ERROR: Incorrect format of starting symbol.
